extensions/amp-lightbox-gallery/0.1/test-e2e/test-open-close.js

```diff
@@ -52,5 +52,24 @@ describes.endtoend(
       await controller.click(closeButton);
       await controller.findElement('amp-lightbox-gallery[hidden]');
     });
+
+    it('should display the image that opened the lightbox', async () => {
+      const clickedImage = await controller.findElement('#basic-2');
+
+      const imageSrc = await controller.getElementAttribute(
+        clickedImage,
+        'src'
+      );
+
+      await controller.click(clickedImage);
+
+      const slideImage = await controller.findElement(
+        `[amp-lightbox-group="default"] .amp-carousel-slide[aria-hidden="false"] img[src="${imageSrc}"]`
```

Comment:
This selector is quite complex, possible to reduce or at the minimum add some text to explain it for future authors?

Reply:
I did add a comment, but oddly enough, it's not that easy to reduce down as we need to wait for this exact element to happen to appear. This element already exists, but not with these exact selection until it is active. Selenium goes just a bit too fast, so we need to be very specific or else we'll select the wrong element and the test will fail.

Reply:
Perfect, makes sense. Thanks for the comment!